src/init_test.py

```diff
@@ -0,0 +1,21 @@
+import build
+import init
+from itertools import product
+from pathlib import Path
+
+
+def test_key_gen(monkeypatch):
+    def mock_build() -> None:
+        print("mock build")
+
+    monkeypatch.setenv("SIGN_BUILDS", "true")
+    monkeypatch.setattr(build, "build", mock_build)
+
+    init.init()
+
+    # Confirm all keys are generated
+    key_names = ["releasekey", "platform", "shared", "media", "networkstack"]
+    key_exts = [".pk8", ".x509.pem"]
+    for k, e in product(key_names, key_exts):
+        path = Path("/srv/keys").joinpath(k).with_suffix(e)
+        assert path.exists()
```

Comment:
maybe put this file in a separate directory "/test"

Reply:
[bb18fb5](https://github.com/lineageos4microg/docker-lineage-cicd/pull/342/commits/bb18fb55287fb17ffc312d0c84d91c69571fb538)